README.md

```diff
@@ -188,6 +188,35 @@ If the Azure identity is in the same resource group as your AKS cluster nodes, y
 az role assignment create --role "Managed Identity Operator" --assignee <sp id> --scope <full id of the managed identity>
 ```
 
+### Disable aad-pod-identity for a specific pod/application
+
+NMI pods modify the nodes' iptables to intercept calls to Azure Instance Metadata endpoint. This means any request that's made to the Metadata endpoint will be intercepted by NMI even if the pod doesn't use aad-pod-identity. `AzurePodIdentityException` CRD can be configured to inform aad-pod-identity that any requests to metadata endpoint originating from a pod that matches labels defined in CRD should be proxied without any processing in NMI. NMI will proxy the request to the metdata endpoint and return the token back as is without any validation.
+
+1. Create the `AzurePodIdentityException` with the same label that will be defined in the pod -
+
+```yaml
+apiVersion: "aadpodidentity.k8s.io/v1"
+kind: AzurePodIdentityException
+metadata:
+  name: test-exception
+spec:
+  PodLabels:
+    foo: bar
+    app: custom
```

Comment:
Can you please put a sample template under aad-pod-identity/deploy/demo or some other location in the code so folks can directly use that file.